lib/src/view/board_editor/board_editor_menu.dart

```diff
@@ -19,6 +27,42 @@ class BoardEditorMenu extends ConsumerWidget {
     return BottomSheetScrollableContainer(
       padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 8.0),
       children: [
+        Padding(
+          padding: Styles.horizontalBodyPadding,
+          child: Wrap(
+            alignment: WrapAlignment.spaceEvenly,
+            children: [
+              SecondaryButton(
+                semanticsLabel: context.l10n.loadPosition,
+                child: Text(context.l10n.loadPosition),
+                onPressed: () {
+                  final notifier = ref.read(editorController.notifier);
+                  pushReplacementPlatformRoute(
```

Comment:
This function doesn't exist anymore. You should update to the last `main` commit. Now we're using directly `Navigator.of(context).push` and we define route as static method of `Screen` type widgets using the `buildScreenRoute` function.